src/js/util.js

```diff
@@ -275,43 +275,25 @@ var Util;
         },
 
         getSelectionRange: function (ownerDocument) {
-            var selection = ownerDocument.getSelection();
-            if (selection.rangeCount === 0) {
-                return null;
-            }
-            return selection.getRangeAt(0);
+            this.deprecated('Util.getSelectionRange', 'Selection.getSelectionRange', 'v5.0.0');
+
+            return Selection.getSelectionRange(ownerDocument);
```

Comment:
Agreed, they definitely belong in Selection.  I don't believe this code will actually fail, since these functions can't be called until all of the js has loaded.  

What I was pointing out though, if you look at the Gruntfile, we are including the JS files in this order:
```javascript 
srcFiles = [
  'src/js/util.js',
  'src/js/defaults/buttons.js',
  'src/js/defaults/options.js',
  'src/js/extension.js',
  ...
]
```

So we'll be calling into Selection from inside Util, even though Util is defined before Selection.  Just something we'll want to remember.